File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue minheap(nums.begin(),nums.end(), greater<int>());

        while(minheap.size()>k){
            minheap.pop();
        }

        return minheap.top();
    }
};

File: Data Structures & Algorithms/subsets/submission-0.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        cursubset, subsets = [], []

        def backtrack(i):
            if i >= len(nums):
                subsets.append(cursubset.copy())
                return
                
            cursubset.append(nums[i])
            backtrack(i + 1)

            cursubset.pop()
            backtrack(i + 1)

        backtrack(0)
        return subsets